src/components/Sidebar/SidebarContent.tsx

```diff
@@ -44,27 +44,34 @@ const renderUserCard = (
     case "recommendations":
       return (
         <ConnectCard
+          key={otherUser.id}
           otherUser={otherUser}
           onViewRouteClick={onViewRouteClick}
         />
       );
     case "favorites":
       return (
         <ConnectCard
+          key={otherUser.id}
           otherUser={otherUser}
           onViewRouteClick={onViewRouteClick}
         />
```

Comment:
Assuming these two returns are the same, then this switch statement can just be simplified.